scss/base/core.scss

```diff
@@ -45,3 +45,7 @@ img,
 iframe {
   border: 0;
 }
+
+.no-link {
```

Comment:
I seem to remember us having a different place for small atomic classes like this. Let's discuss - also why we need this class.